packages/visualizations/src/components/Table/Cell/Format/GeoFormat.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
+    import { tick } from 'svelte';
+    import { WebGlMap } from 'components/Map';
+    import tippy from 'components/utils/tippy';
+
+    import type { WebGlMapOptions, WebGlMapData } from 'components/Map';
+
+    export let rawValue: unknown;
+    export let display = (v: unknown) => v;
+    export let mapOptions: WebGlMapOptions;
+    export let sources: (v: unknown) => WebGlMapData['sources'] = () => ({});
+    export let layers: (v: unknown) => WebGlMapData['layers'] = () => [];
+
+    let tooltipContent;
+    let showMap;
+
+    $: data = {
+        sources: sources(rawValue),
+        layers: layers(rawValue),
+    };
+</script>
+
+<div
+    role="button"
+    tabindex="0"
+    use:tippy={{
+        content: tooltipContent,
+        theme: 'ods-visualization-table',
+        delay: [500, 0],
+        duration: [275, 0],
+        maxWidth: 'none',
+        onTrigger: (instance) => {
```

Comment:
Do you need instance as an arg here ?

Reply:
nop, fixed here 
https://github.com/opendatasoft/ods-dataviz-sdk/pull/245/commits/e7e8a4ed09953ea970f68dcb6db8144c45897773